docs/consensus.md

```diff
@@ -0,0 +1,46 @@
+# NEO-GO consensus node
+
+Neo-go node can act as a consensus node.
+To make this possible go dBFT implementation from [nspcc-dev/dbft](https://github.com/nspcc-dev/dbft) is used.
```

Comment:
`Neo-go node can act as a consensus node. It uses pure Go dBFT implementation from ... to make it possible.` ?